src/app/shhhhh/ShhhhhLandingPage.tsx

```
@@ -0,0 +1,491 @@
+
+import Image from 'next/image'
+import { useEffect, useRef, useState } from 'react'
+import { useRouter } from 'next/navigation'
+import { AnimatePresence, motion } from 'framer-motion'
+import { useQueryClient } from '@tanstack/react-query'
+import posthog from 'posthog-js'
+import { Button } from '@/components/0_Bruddle/Button'
+import { Marquee } from '@/components/LandingPage'
+import { useAuth } from '@/context/authContext'
+import { PixelatedCardFace } from '@/components/Card/share-asset/PixelatedCardFace'
+import { Sparkle, Star } from '@/assets'
```

ESLint

[error] '`@/assets`' import is restricted from being used. Import from a specific file instead of the '`@/assets`' barrel — barrels force the bundler to load every re-export and hurt build perf. See CLAUDE.md.

(no-restricted-imports)